### DISCUSSION TITLE
Dipolo

### DISCUSSION THREAD

**Discussion by DimisMeu** [iptv-org/community#435] [Show and tell]
Add this Greek Channel: Dipolo
Stream: https://channel.streams.ovh:1936/dipolo/dipolo/playlist.m3u8

----------

https://channel.streams.ovh:2020/public/dipolo
https://sfiga.de/tv/

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel
